### PR TITLE
Fixed Comma Errors + Capitalization Inconsistencies.\

### PR DESCRIPTION
**Pull request by JesseSumrak**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Fixed Comma Errors + Capitalization Inconsistencies

### Full changelog

* [Edited docs to ...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/0.14.x/getting-started/quickstart.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ title: 5-minute Quickstart
   <a href="https://konghq.com/install/">installed Kong</a> &mdash; It should only take a minute!
 </div>
 
-In this section, you'll learn how to manage your Kong instance. First we'll
+In this section, you'll learn how to manage your Kong instance. First, we'll
```
Comment (coopr) [inline app/0.14.x/getting-started/quickstart.md]: This same slight error occurs in prior versions of this doc (like https://github.com/Kong/docs.konghq.com/blob/master/app/0.13.x/getting-started/quickstart.md ) - please make this correction in all versions of the docs.

File: app/0.14.x/getting-started/quickstart.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Issue the following command to [reload][CLI] Kong without downtime:
 $ kong reload
 ```
 
-## Next Steps
+## Next steps
```
Comment (coopr) [inline app/0.14.x/getting-started/quickstart.md]: This capitalization inconsistency also occurs in prior docs versions - please correct them all.

**Comment by shinypenny31** [comment]
Will do!

**Comment by coopr** [comment]
Please combine all your contributions into one PR, please.

**Pull request closed without merging**